src/pages/projects.js

```diff
@@ -1,7 +1,39 @@
 import React from "react";
 
 import Layout from "../components/layout";
+import ProjectCard from "../components/ProjectCard";
 
-const Projects = () => <Layout>Future components HERE</Layout>;
+import { projectsData } from "../db/projects";
+import "../styles/pages/projects.scss";
+
+const projectsGrid = projectsData.projectsList.map((project) => (
+    <ProjectCard project={project} key={project.photoAlt} />
+));
+
+const projectsList = projectsData.offerList.map((item) => (
+    <li key={item.slice(0, 33)} className="projects__paragraph">
+        {item}
+    </li>
+));
```

Comment:
Separate components?

Reply:
I was thinking about making separate directory for controller files but this app is to small for this kind of logic. If it grow bigger then it might be implemented.